setup.py

```diff
@@ -42,35 +42,36 @@ def get_long_description():
     "rich-argparse",
     "tqdm",
     "simple-term-menu",
+    "pydantic>=1.10.10,<2.0.0",
+    "pydantic-duality>=1.2.4",
+    "websocket-client",
+    "watchfiles",
```

Comment:
Isn't it an optional uvicorn (that is, server) dependency for hot code reloading?